docs/Job-reference.md

```diff
@@ -5154,13 +5154,15 @@ job('example-2') {
 job {
     publishers {
         rundeck(String jobId) {
-            options(Map<String, String> option)           // defaults to empty map if omitted
+            options(Map<String, String> option)             // defaults to empty map if omitted
             option(String key, String value)
-            tag(String tag)                               // defaults to empty string if omitted
-            nodeFilters(Map<String, String> filters)      // defaults to empty map if omitted
+            tag(String tag)                                 // defaults to empty string if omitted
+            nodeFilters(Map<String, String> filters)        // defaults to empty map if omitted
             nodeFilter(String key, String value)
-            shouldWaitForRundeckJob(boolean value = true) // defaults to false if omitted
-            shouldFailTheBuild(boolean value = true)      // defaults to false if omitted
+            shouldWaitForRundeckJob(boolean value = true)   // defaults to false if omitted
+            shouldFailTheBuild(boolean value = true)        // defaults to false if omitted
+            shouldIncludeRundeckLogs(boolean value = true)  // defaults to false if omitted - since 1.35
+            shouldWaitForRundeckJobAndIncludeRundeckLogs(boolean value = true)  // sets both options as shouldIncludeRundeckLogs requires shouldWaitForRundeckJob - since 1.35
```

Comment:
do not include shortcut methods, just set shouldWaitForRundeckJob when setting shouldIncludeRundeckLogs